src/components/song-view/song-view.tsx

```diff
@@ -57,9 +57,9 @@ export function SongView({ song }: Readonly<SongViewProps>) {
           <div className="space-y-2">
             <div className="text-lg font-bold">INTRODUCTION</div>
             <div key="introduction">
-              <div>{convertJsonToComponent(song?.introduction ?? '{}')}</div>
+              <div>{convertWysiwygToComponent(song?.introduction ?? '{}')}</div>
```

Comment:
Probably best to switch these defaults to empty strings i.e. `'{}'` -> `''` 
so that they are caught by the first if statement in the convert method.

Reply:
Good catch